CourseSketchProjects/CourseSketchDatabaseServer/src/main/java/database/institution/mongo/BankProblemManager.java

```diff
@@ -361,8 +360,8 @@ public static boolean mongoUpdateBankProblem(final Authenticator authenticator,
     public static String mongoGetRegistrationKey(final Authenticator authenticator, final DB database,
             final String authId, final String bankProblemId)
             throws AuthenticationException, DatabaseAccessException {
-        final DBRef myDbRef = new DBRef(database, PROBLEM_BANK_COLLECTION, convertStringToObjectId(bankProblemId));
-        final DBObject cursor = myDbRef.fetch();
+        final DBCollection collection = database.getCollection(getCollectionFromType(School.ItemType.BANK_PROBLEM));
```

Comment:
please change the name to problemBank or problemBankCollection or problemCollwction